[2022-03-18 20:56:26,713] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.task_4 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 20:56:26,758] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.task_4 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 20:56:26,758] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 20:56:26,759] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2022-03-18 20:56:26,761] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 20:56:26,792] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): task_4> on 2022-03-17T00:00:00+00:00
[2022-03-18 20:56:26,799] {standard_task_runner.py:52} INFO - Started process 5936 to run task
[2022-03-18 20:56:26,806] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'task_4', '2022-03-17T00:00:00+00:00', '--job-id', '40', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmp3xnyu83w', '--error-file', '/tmp/tmpkedv5kb_']
[2022-03-18 20:56:26,810] {standard_task_runner.py:77} INFO - Job 40: Subtask task_4
[2022-03-18 20:56:26,943] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_dag.task_4 2022-03-17T00:00:00+00:00 [running]> on host pc
[2022-03-18 20:56:27,182] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T00:00:00+00:00
[2022-03-18 20:56:27,207] {logging_mixin.py:104} INFO - choose best model None
[2022-03-18 20:56:27,207] {python.py:151} INFO - Done. Returned value was: None
[2022-03-18 20:56:27,235] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=task_4, execution_date=20220317T000000, start_date=20220318T152626, end_date=20220318T152627
[2022-03-18 20:56:27,323] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-18 20:56:27,350] {local_task_job.py:151} INFO - Task exited with return code 0
